.github/workflows/test.yml

```diff
@@ -45,6 +45,10 @@ jobs:
           if pip-licenses | grep -E -v 'Artistic License|LGPL|Public Domain' | grep GNU; then echo 'Please avoid introducing *GPL dependencies'; false; fi
       - uses: tj-actions/changed-files@v45
         id: file_changes
+        with:
+          # Only run pre-commit on EV files themselves, not on submodules
+          # See https://github.com/EveryVoiceTTS/EveryVoice/issues/555
+          exclude_submodules: true
```

Comment:
I'm going to be ask that this be made a separate commit / PR as this is not addressing the `text-to-spec` confusion but rather arose from fixing the confusion.

Reply:
CI fixes separated into #557